plothist/hep_plotters.py

```diff
@@ -141,21 +145,47 @@ def compare_data_mc(
 
     # Ignore divide-by-zero warning
     np.seterr(divide="ignore", invalid="ignore")
-    # Compute data/MC ratio
-    ratio = np.where(
-        mc_hist_total.values() != 0, data_hist.values() / mc_hist_total.values(), np.nan
-    )
-    # Compute scaled uncertainties
-    scaled_data_uncertainty = np.sqrt(data_hist.variances()) / data_hist.values()
-    scaled_mc_uncertainty = np.sqrt(mc_hist_total.variances()) / mc_hist_total.values()
+    if ratio == "diff":
+        # Compute data/MC ratio
+        ratio_values = np.where(
+            mc_hist_total.values() != 0,
+            data_hist.values() / mc_hist_total.values(),
+            np.nan,
+        )
+        # Compute scaled uncertainties
+        scaled_data_uncertainty = np.sqrt(data_hist.variances()) / data_hist.values()
```

Comment:
Looks wrong to me, we should divide by `mc_hist_total.values()` I think

Reply:
```
RuntimeWarning: overflow encountered in double_scalars
  delta = (x1t - x0t) * margin
```

Reply:
I do not understand the cause of this warning

Reply:
Dividing by `np.nan` or `0` caused this.
Added a `np.where()` to get rid of this problem.